api/src/test/java/org/openmrs/api/FormServiceTest.java

```diff
@@ -46,6 +38,13 @@
 import org.openmrs.util.DateUtil;
 import org.openmrs.util.OpenmrsConstants;
 
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
```

Comment:
Looks like just formatting changes too in this class?

Reply:
Indeed, apologies for the noise. A couple of tests had occasionally failed in the build and I re-ran them in the IDE, that probably led to the changes.

Reply:
Should be back to the state it had on master now.